addons/purchase_requisition/static/src/js/purchase_order_compare_list_renderer.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import ListRenderer from 'web.ListRenderer';
+
+
+const PurchaseOrderCompareListRenderer = ListRenderer.extend({
+    init: function (parent, state, params) {
+        this._super(...arguments);
+        this.active_id = parent.res_id;
+    },
+
+    /**
+     * @override
+     * @private
+     * @returns {Promise}
+     */
+     _renderRow: function (record) {
+        const $tr = this._super(...arguments);
+        if (record.res_id === this.active_id && $tr.find('o_list_record_remove')) {
```

Comment:
It's missing a dot before the class name, and also, it works even it never find the remove button because the an empty jQuery object is a truly value (it seems calling `remove` on an empty jQuery object doesn't cause any error) 
```suggestion
        if (record.res_id === this.active_id && $tr.find('.o_list_record_remove').length) {
```